src/resmom_win/start_exec.c

```diff
@@ -2965,14 +2992,13 @@ start_process(task *ptask, char **argv, char **envp, bool nodemux)
 				argv_out[0] = progname_out;	
 	
 				argv_str = str_array_to_str(argv_out,
-								" ");
+								' ');
```

Comment:
Add this to the end of the previous line.

Reply:
Done.